packages/core/src/Toast/ToastViewport.vue

```diff
@@ -172,9 +172,8 @@ function getSortedTabbableCandidates({ tabbingDirection }: { tabbingDirection: '
   >
     <FocusProxy
       v-if="hasToasts"
-      :ref="(node: ComponentPublicInstance) => {
-        headFocusProxyRef = unrefElement(node) as HTMLElement
-        return undefined
+      :ref="(node) => {
```

Comment:
Not sure how to feel about these refs; would perhaps be better to keep the raw ref and unwrap when needed..? 🤔